src/shared/components/ncTable/partials/TableCellNumber.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<div>
-		{{ column.numberPrefix }}{{ getValue }}{{ column.numberSuffix }}
+		{{ column.numberPrefix }}&nbsp;{{ getValue }}{{ column.numberSuffix }}
```

Comment:
Should we also separate the suffix?

Reply:
Totally right, it depends on the use case. There are examples for all combinations: 
- Currency prefix without spaces `$100`
- Severity level prefix with spaces `Level 1`
- Percentage suffix without spaces `75%`
- Currency suffix with spaces `100 USD`

I think we can remove the space from both prefix and suffix because it can always be added by the person who is configuring the table, but if there is a space by default it could be frustrating to not be able to remove it :)